corona-warn-app/cwa-app-android
# Create automated app screenshots - Submission flow QR-Code/TAN (EXPOSUREAPP-2997)

screenshots for submission flow

### Testing:

~~After targeting Android 11, MockK fails to run on API 11 when spy is used. For now please run the tests on pre-Android 11
mockk/mockk#519~~ (No longer necessary after target API downgrade)

1. install [fastlane](https://docs.fastlane.tools/getting-started/ios/setup/)
1. run `./gradlew clean assembleDebug assembleAndroidTest` for clean state
1. run `bundle exec fastlane screengrab` to execute tests
